### ISSUE TITLE
Recommended System Requirements

### ISSUE THREAD

**Issue by yihe-61wu** [StanfordVL/iGibson#236]
Hi, my research project will be using the simulator, and I have managed to install almost everything, run the demos, and test benchmarks on an old Linux machine borrowed from a colleague (CPU: 8 core i7; GPU: Nvidia with ~8G VRAM). The project has funding for me to buy a new, dedicated machine. I'm wondering how much improvement a potential upgrade in hardware would bring me, compared to the minimal requirements or the current model I'm using (happy to provide more details).

**tl;dr** How does the simulation efficiency scale with hardware?

---

Apologise if the question is stupid; this is my first time working with any simulator that requires GPU support!
Thank you in advance!
